content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,173 @@
+# Build and Run `rippled` on Ubuntu
+
+`rippled` is the core peer-to-peer server that manages the XRP Ledger. A `rippled` server can connect to a network of peers, relay cryptographically signed transactions, and maintain a local copy of the complete shared global ledger.
+
+For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
+
+Use these instructions to build a `rippled` binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. These instructions were tested on Ubuntu 16.04 LTS.
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+Here are the **minimum** system requirements for building and running `rippled`.
+
+**_To build rippled:_**
+
+* RAM: 8GB
+
+**_To run rippled:_**
+
+* RAM: 4GB
+* CPU: 64-bit x86_64, 2+ cores
+* Disk: 50GB SSD with 500 IOPS for the database partition
+* Network: A fast network connection is preferable
+
+If you are running `rippled` in an Amazon EC2 instance, Ripple recommends selecting a **Type** of **_m3.large_**. If you choose to run a validating `rippled` server, you may need more resources.
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+```
+sudo apt-get update
+```
+2. Upgrade currently installed packages.
+```
+sudo apt-get -y upgrade
+```
+3. Install Git.
+```
+sudo apt-get -y install git
+```
+4. Install SCons.
+```
+sudo apt-get -y install scons
+```
+5. Install Ctags.
+```
+sudo apt-get -y install ctags
+```
+6. Install `pkg-config`.
+```
+sudo apt-get -y install pkg-config
+```
+7. Install Protocol Buffers.
+```
+sudo apt-get -y install protobuf-compiler
+sudo apt-get -y install libprotobuf-dev
+```
+8. Install Secure Socket Layer (SSL) toolkit development files.
+```
+sudo apt-get -y install libssl-dev
+```
+9. Install `python-software-properties`.
+```
+sudo apt-get install -y python-software-properties
+```
+10. Install Boost.
+```
+sudo add-apt-repository -y ppa:boost-latest/ppa
+sudo apt-get -y update
+sudo apt-get install -y libboost-all-dev
+```
+11. Get `rippled` source code.
+```
+git clone https://github.com/ripple/rippled.git
+cd rippled
+git checkout master
+```
+12. Build `rippled` binary executable from source code. This may take about 30 minutes.
+
+    SCons saves the built executable in `rippled/build`.
+
+```
+scons
+```
+
+
+## 2. Configure `rippled`
+
+1. Configure `rippled.cfg`.
```

Comment:
This is GREAT. Thank you - yes, it would be great to describe the hierarchy of where `rippled` looks for the config files. I'll open a ticket (witness the self-control... lol) 

I'll put this in the ticket, but just because I think you might be interested - for this task, when I run `rippled` with no config files present, this is the error message: 

![image](https://user-images.githubusercontent.com/5481259/34171567-9a2b193c-e4a3-11e7-8fad-d84f1053fd79.png)

This is why I put the config files in that location. 😂  Perhaps this is the wrong location for it to first look for the config files if we recommend that users run `rippled` as a non-root users?

Reply:
When you use `sudo` it runs it as root, so it looks in the root directory.